services/github/src/github-auth.js

```diff
@@ -24,16 +24,27 @@ const retryPlugin = (octokit, options) => {
 };
 const Octokit = require('@octokit/rest').plugin([retryPlugin]);
 
-module.exports = async ({cfg}) => {
-  const keyRe = /-----BEGIN RSA PRIVATE KEY-----\n.*\n-----END RSA PRIVATE KEY-----\n?/;
-  if (!keyRe.test(cfg.github.credentials.privatePEM)) {
-    throw new Error(`Malformed GITHUB_PRIVATE_PEM: must match ${keyRe}`);
+const getPrivatePEM = cfg => {
+  const keyRe = /-----BEGIN RSA PRIVATE KEY-----(\n|\\n).*(\n|\\n)-----END RSA PRIVATE KEY-----(\n|\\n)?/;
+  const privatePEM = cfg.github.credentials.privatePEM;
+  if (!keyRe.test(privatePEM)) {
+    throw new Error(`Malformed GITHUB_PRIVATE_PEM: must match ${keyRe}; ` +
+      `got a value of length ${privatePEM.length}`);
```

Comment:
Thanks! This will be handy for operators to tell whether the problem is with a wrong value or a blank value getting through, without a risk of leaking the contents.